src/test/java/com/kodekonveyor/market/project/ProjectModelDTOTestData.java

```diff
@@ -4,16 +4,16 @@
 
 import javax.annotation.Generated;
 
-import com.kodekonveyor.market.tasks.TaskTestData;
+import com.kodekonveyor.market.tasks.TaskDTOTestData;
 
 @Generated("by zenta-tools")
 public class ProjectModelDTOTestData {
 
   public final static ProjectModelDTO get() {
     final ProjectModelDTO projectModelDTO = new ProjectModelDTO();
     projectModelDTO.setId(ProjectModelTestData.ID);
-    projectModelDTO.setTask(Set.of(TaskTestData.ID));
-    projectModelDTO.setMilestone(Set.of(MilestoneTestData.ID));
+    projectModelDTO.setTask(Set.of(TaskDTOTestData.get()));
```

Comment:
If you create a test case with multiple different tasks your production code would fail. Please write that test case.